tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "rootDir": "src",
     "module": "commonjs",
     "noImplicitAny": true,
-    "strictNullChecks": true,
+    "strictNullChecks": false,
```

Comment:
Instead of making this false, can you update `src/firebase-rest.ts:123` to:

```typescript
if (res.statusCode && Math.floor(res.statusCode / 100) !== 2)
```